vispy/visuals/mesh_normals.py

```diff
@@ -0,0 +1,61 @@
+# -*- coding: utf-8 -*-
+# -----------------------------------------------------------------------------
+# Copyright (c) Vispy Development Team. All Rights Reserved.
+# Distributed under the (new) BSD License. See LICENSE.txt for more info.
+# -----------------------------------------------------------------------------
+
```

Comment:
I'm not sure why the linter doesn't complain about this, but since I've asked for other edits could you remove this blank line.

Reply:
For info, there is this blank line in quite a few other files in the same directory.

Reply:
Yeah...I'm trying to standardize things and just haven't gotten around to adding a auto-formatter like `yapf` or `black`. We also have so many "wrong" docstrings that I had to disable to docstring testing because the updated version of the test was failing for a lot of vispy docstrings.